Frontend/src/data/item/onUpdate.js

```diff
@@ -5,7 +5,13 @@ import columns from "./columns";
 import { setNumericValuesDefault0 } from "../utils";
 import Logger from "js-logger";
 
-export default async (item, closePopup) => {
+export default async (item, closePopup, formRef) => {
```

Comment:
You could think about reusing this code somehow because I think we have this exact same function in 3 places now.